src/components/molecules/widgets/KilledAndInjuredCountPerAgeGroupWidget.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+import { IWidgetKilledAndInjuredCountPerAgeGroup } from 'models/WidgetData';
+import { MultiBarChart } from '../GenericBarChart';
+import { convertItemsToBarSeries } from 'utils/barChart.utils';
+
+interface IProps {
+  data: IWidgetKilledAndInjuredCountPerAgeGroup;
+}
+
+const KilledAndInjuredCountPerAgeGroupWidget: FC<IProps> = ({ data }) => {
+  const { items, text } = data;
+  const multiBarSeries = convertItemsToBarSeries(items);
+  return <MultiBarChart isStacked={false} isPercentage={false} data={multiBarSeries} textLabel={text.title} />;
```

Comment:
I guess we need to pass the subtitle that you need instead of passing just `text.title`

Reply:
Passed a subtitle as an optional argument